src/figuresClasses.ts

```diff
@@ -1,19 +1,108 @@
 export interface Figure {
+  color: string;
+  shape: string;
 
+  getArea(): number;
 }
 
-export class Triangle {
+function isTrianglePossible(a: number, b: number, c:number): boolean {
+  const sum = [a, b, c];
+  const largest = Math.max(a, b, c);
```

Comment:
If you create `sum` array before why not to use destructuring: `const largest = Math.max(...sum);`